cime_config/stream_cdeps.py

```diff
@@ -716,11 +716,11 @@ def _sub_paths(
                             date_string = (year_format + "-{:02d}-{:02d}").format(
                                 adjusted_year, adjusted_month, adjusted_day
                             )
-                            new_file = line.replace(match.group(0), date_string)
-                            if os.path.exists(new_file):
-                                new_lines.append(new_file)
+                            new_line = line.replace(match.group(0), date_string)
+                            if os.path.exists(new_line):
+                                new_lines.append(new_line)
                             else:
-                                print(f"   WARNING:not adding missing file {new_file}")
+                                print(f"   WARNING:not adding missing file {new_line}")
```

Comment:
Should this message be changed from "missing file" to "missing line"?

Reply:
It actually should be new_line - where new_line actually represents the file that is looked for. I can try to update this more generally in the next cdeps PR coming up soon.